Makefile

```diff
@@ -198,12 +198,6 @@ create-updates.img: bots
 	-rm $(UPDATES_IMG)
 	make $(UPDATES_IMG)
 
-# test runs in kernel_t context and triggers massive amounts of SELinux
-# denials; SELinux gets disabled, but would still trigger unexpected messages
-# we create huge VMs, so we need to reduce parallelism on CI
-integration-test: prepare-test-deps test/reference $(UPDATES_IMG)
```

Comment:
This means there's no longer a single `make ...` which you can use in a local checkout to run integration tests. But I figure if you wanted that, inspecting or running `test/run` should actually do fine.